### ISSUE TITLE
Empty Folder not handled

### ISSUE THREAD

**Issue by UsmanMohammad** [Geertvdc/VSTS-Build-AzureWebAppVFS#2]
`2017-05-11T08:07:51.3708387Z Azure Initialized
2017-05-11T08:07:51.5113128Z VFS scripts Initialized
2017-05-11T08:08:00.5105695Z Retrieved web app: Microsoft.Azure.Management.WebSites.Models.Site in Resource group: Web-Apps-Stage-Luxtripper
2017-05-11T08:08:00.5105695Z Retrieving publishing profile
2017-05-11T08:08:01.5791361Z Publishing profile retrieved
2017-05-11T08:08:01.5801334Z Remove-FileFromWebApp path: /
2017-05-11T08:08:01.5801334Z Recursive delete so Get-FileListFromWebApp to see which files to delete: /
2017-05-11T08:08:02.0048559Z ##[error]You cannot call a method on a null-valued expression.
2017-05-11T08:08:02.0228538Z ##[section]Finishing: Delete All files from Azure Web App
`
The extension doesn't handle null array value when the specified directory is already empty.

**Comment by jonathan-martin-endava** [comment]
@UsmanMohammad  Yep we had this problem as well, using v2.0.0. Quick fix is to update line 42 of vfs.psm1 within the _tasks folder on the agents with
                       return $null  # Write-Output "File not found (but ignored because of setting)"
The Get-FileListFromWebApp() function returns the Write-Output message into the variable used to store found dirs; subsequently, in the ForEach loop, it dies at the SubString method call on a null object.

> $file = "File not found (but ignored because of setting)"
> ($file.href).SubString(0)
You cannot call a method on a null-valued expression.
At line:1 char:1
+ ($file.href).SubString(0)
+ ~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (:) [], RuntimeException
    + FullyQualifiedErrorId : InvokeMethodOnNull

> $file = @{href="some-directory"}
> ($file.href).SubString(0)
some-directory
